lib/krikri/parsers/json_parser.rb

```diff
@@ -44,7 +44,32 @@ def values?
 
       private
 
-      def get_child_nodes(name)
+      ##
+      # Return a Krikri::Parser::ValueArray of child nodes that match a name
+      # or "name expression" of exclusive names separated by "|" characters.
+      #
+      # The values come from the property that comes first in the JSON object,
+      # which is given in the name expression.  This behavior maintains
+      # consistency with Krikri::XmlParser#get_child_nodes, which takes the
+      # first value in the document due to way in which XPATH is employed.
+      #
+      # @see Krikri::XmlParser#get_child_nodes
+      #
+      # @example
```

Comment:
I think it would be a good idea to revisit some of this documentation. It seems like some of it should sit in the base `Krikri::Parser` abstract method. The specific implementations should point back to that, and add any specific details like " The values come from the property that comes first in the JSON object which is given in the name expression...".